plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLocalConcurrentWritesTest.java

```diff
@@ -196,6 +196,12 @@ private void testConcurrentInsertsSelectingFromTheSameTable(boolean partitioned)
             // T1: (1, 10)
             // T2: (2, 10)
             // T3: (3, 10)
+            // However, because the isolation level is WriteSerializable, it may likely happen that competing
+            // queries read will commit successfully by reading the same version.
+            // This can look like e.g.:
+            // T1: (1, 10)
+            // T2: (1, 10)
```

Comment:
This sequence cannot happen in WriteSerializable, as there is no sequence of:
`INSERT INTO " + tableName + " SELECT COUNT(*), 10 AS part FROM " + tableName`
that could produce above result when applied in sequence.